src/main/java/com/atlassian/shele/shelePlugin/rest/RestIssueService.java

```diff
@@ -1,51 +1,52 @@
 package com.atlassian.shele.shelePlugin.rest;
 
 
-import com.atlassian.activeobjects.external.ActiveObjects;
-import com.atlassian.jira.issue.Issue;
-import com.atlassian.sal.api.transaction.TransactionCallback;
-import com.atlassian.shele.shelePlugin.ao.IssueEntity;
-import net.java.ao.EntityManager;
-import net.java.ao.RawEntity;
-
-import javax.inject.Inject;
-import javax.ws.rs.*;
+import com.atlassian.jira.component.ComponentAccessor;
+import com.atlassian.shele.shelePlugin.ao.ProjectDTO;
+import com.atlassian.shele.shelePlugin.ao.ProjectEntity;
+import com.atlassian.shele.shelePlugin.ao.ProjectEntityService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
-
-import java.beans.PropertyChangeListener;
-import java.time.LocalDateTime;
-import java.util.Locale;
-
-import static com.google.gson.internal.$Gson$Preconditions.checkNotNull;
+import java.util.List;
 
 @Path("/api")
 @Consumes(MediaType.APPLICATION_JSON)
 @Produces(MediaType.APPLICATION_JSON)
+@Component
+@Slf4j
 public class RestIssueService {
 
-    private  ActiveObjects activeObjects;
+    private final ProjectEntityService persistLayer;
 
-    public  RestIssueService(ActiveObjects activeObjects) {
-        this.activeObjects = checkNotNull(activeObjects);
+    @Autowired
+    public  RestIssueService(ProjectEntityService persistLayer) {
+        this.persistLayer=persistLayer;
     }
-        @GET
-        @Path("/getIssue")
-        public Response createIssue(){
-            IssueEntity is = activeObjects.create(IssueEntity.class);
-            is.setIssueId(2);
-            is.setAuthorIssue("Jessy");
-            is.save();
-           return Response.ok().build();
-        }
- 
-
-        
-        
-
-
-
 
+        @POST
+        @Path("/issue")
+        public Response.Status submitProject(ProjectDTO dto){
+            ProjectEntity projectEntity = persistLayer.getEntity();
+            persistLayer.deleteRaws();
+            List<String> list = dto.getProject();
+            List<Long> longs = dto.getEventTypeIds();
+                for(String s : list){
+                    ProjectEntity project =persistLayer.getEntity();
+                    project.setProject(s);
+                    project.setEventTypeId(longs.toString());
+                   project.setIssueStatus(ComponentAccessor.getWorkflowManager().getDefaultWorkflow().getLinkedStatusObjects().get(0).getSimpleStatus().getName());
+                   project.save();
```

Comment:
almost the same logic is performed in the `ProjectEntityService::getEntity()`, please perform all the logic in the service